app/modules/transcribe/annotations/annotations.factory.js

```diff
@@ -57,8 +60,36 @@ function AnnotationsFactory(localStorageService) {
         return annotation;
     }
 
+
+    function checkVariants(annotation) {
+        //splits string by whitespace (different encodings)
+        var lemmas = annotation.text.split(/\s|&nbsp;/g);
```

Comment:
Why `\s` and `&nbsp;`? Also, are you stripping out any punctuation? I'm guessing there are places where periods have actual meaning, but commas etc?